Containerfile

```diff
@@ -151,8 +151,8 @@ ansible-galaxy collection install -v -f -r /ansible/galaxy/requirements.yml -p /
 ln -s /usr/share/ansible/collections /ansible/collections
 
 # prepare project repository
-if [ "$OPENSTACK_VERSION" = "master" ]; then git clone https://github.com/openstack/kolla-ansible /repository; fi
-if [ "$OPENSTACK_VERSION" != "master" ]; then git clone -b stable/$OPENSTACK_VERSION https://github.com/openstack/kolla-ansible /repository; fi
+if [ "$OPENSTACK_VERSION" = "master" ]; then git clone https://opendev.org/openstack/kolla-ansible.git /repository; fi
```

Comment:
I would stick to the shorter URLs without the `.git` suffix, they should be functionally identical.